R/query.R

```diff
@@ -25,14 +25,14 @@ query <- function(language, source) {
 #' captures of the query for further use. Both functions return the same
 #' information, just structured differently depending on your use case. 
 #' 
-#' - `query_matches()` returns the captures first grouped by _predicate_, and
-#'   further grouped by _match_ within each predicate. This is useful if you
-#'   include multiple predicates in your query or if you expect multiple
+#' - `query_matches()` returns the captures first grouped by _pattern_, and
+#'   further grouped by _match_ within each pattern. This is useful if you
+#'   include multiple patterns in your query or if you expect multiple
 #'   captures per match.
 #' 
 #' - `query_captures()` returns a flat list of captures ordered by their node
 #'   location in the original text. This is normally the easiest structure to
-#'   use if you have a single predicate with one capture per match.
+#'   use if you have a single pattern with one capture per match.
```

Comment:
Do you really mean "one capture per match"? I feel like `query_captures()` was useful to me in #3 with a single match but with multiple captures. Maybe the point is that the situation needs to be simple enough for the flat structure to be unambiguous?